mimic/model/nova_objects.py

```diff
@@ -739,6 +760,10 @@ def request_list(self, http_get_request, include_details, absolutize_url,
         """
         to_be_listed = self.servers
 
+        if changes_since is not None:
+            since = timestamp_to_seconds(changes_since)
+            to_be_listed = filter(lambda s: s.update_time > since, to_be_listed)
```

Comment:
No I'm crazy, sorry, we can test with Nova.  :D  It seems to be `>=` (I made a change, took the update time, used that as the `changes-since` param, and it returned that one server)